### ISSUE TITLE
PA Review: User Authentication

### ISSUE THREAD

**Issue by ElishaPark** [ElishaPark/dancey-plus#7]
- [x] Backend: DB, model, controller, views
- [x] Redux Loop: ajax, actions, reducer
- [x] Presentational Components and Containers
- [x] Styling
- [x] Smooth, bug-free navigation
- [x] Adequate and appropriate seeds

**Comment by linakherchi** [comment]
Thank you Elisha ! 

I liked your logo update :) 

Couple of comments regarding your first feature: 

- I see that you have a footer in the splash page - Would be a great idea if you could add some links redirecting the visitor to your Github / Linkedin profiles
- As I click on login, I feel that the size of the page where I go (SessionForm) is much bigger. Also, there is no space between the bottom of the form and the bottom of the page. Is it possible to decrease the size of the form ? Keep the footer in this page as well ? Feel free to re-organize it as what feels the most intuitive to you.
- I do not have a button to log me out after I log in/ sign up 

Great job !

**Comment by ElishaPark** [comment]
Hey Lina- these features should be visible now (and hopefully the sizing isn't off like the last time you viewed). If it looks weird, it might also be the zoom factor on your browser that's affecting it? I realized one day that I was zoomed in more on the page than normal, which was throwing off the sizing. Thanks!

**Comment by linakherchi** [comment]
Looking very clean ! And I enjoyed the 0$ for subscription :) Wish it was true.

Concerning user auth, it looks great ! You just need to make sure that errors are cleared when moving from a form to another after an unsuccessful account creation. (e.g. I tried to login with an account that did not exist and got errors. When I clicked on signup the errors from my unsuccessful login were still present.)
